changelog.d/2-features/pr-2076

```diff
@@ -0,0 +1 @@
+A guest link cannot be generated if the conversation access role doesn’t allow guests.
```

Comment:
```suggestion
Enforce conv access roles more tightly on the backend: if a guests are not allowed, block guest link creation (new behavior) as well as ephemeral users joining (old behavior).
```

I would also put this either in bug fixes (it's not really a feature) or in release notes (it's not really a bug, just something to be aware of in general, just in case).  What do you think?